docs/UserGuide.md

```diff
@@ -534,3 +587,37 @@ If your changes to the data file makes its format invalid, SoConnect will discar
 | **Delete Todo** | `todo delete INDEX` <br> e.g. `todo delete 3`                                                                                                                                                                        |
 | **Clear Todo**  | `todo clear`                                                                                                                                                                                                         |
 | **Show Todo**   | `todo show`<br> `todo show today` <br> `todo show date/DATE` <br> `todo show date/DATE to DATE` <br> `todo show t/TAG` <br> `todo show pr/Priority` <br> e.g. `todo show`, `todo show pr/high`, `todo show t/CS2100` |
+
+--------------------------------------------------------------------------------------------------------------------
+
+## 10. Glossary
```

Comment:
The spacing between each glossary definition seems to be excessive? I don't reckon that we should use headers for the definitions to reduce the spacing. Additionally, making them headers causes them to appear in the TOC which might not be desirable

Reply:
Yes I recognize that. I was actually torn because I want to keep the consistency (big header - sub header - etc), but also notice that it might be excessive. But yes, generally should be no problem to have the glossary words as normal words.

Reply:
Yup, the spacing is a bit excessive, either we follow the one in DG or we can create a table for it.